storage/store/inmemory/unsynchronized/collections.go

```diff
@@ -84,7 +84,7 @@ func (c *Collections) LightByTransactionID(txID flow.Identifier) (*flow.LightCol
 
 // Store inserts the collection keyed by ID and all constituent transactions.
 // No errors are expected during normal operation.
-func (c *Collections) Store(collection *flow.Collection) (flow.LightCollection, error) {
+func (c *Collections) Store(collection *flow.Collection) (*flow.LightCollection, error) {
```

Comment:
I think it has to be how it was before having a `LightCollection` as value instead of pointer. Structurally it contains a single pointer, there is no need to allocate it on heap once more. Having a value will be faster and with less memory and GC overhead.